content/influxdb/v2.2/reference/cli/influx/export/all.md

```diff
@@ -45,6 +45,26 @@ influx export all [flags]
 |      | `--skip-verify`   | Skip TLS certificate verification                                                               |             | `INFLUX_SKIP_VERIFY`  |
 | `-t` | `--token`         | API token                                                                                       | string      | `INFLUX_TOKEN`        |
 
+## Resources
+The following resources can be exported:
+
+- Bucket                       
+- Check                        
+- CheckDeadman                 
+- CheckThreshold               
+- Dashboard                    
+- Label                        
+- NotificationEndpoint         
+- NotificationEndpointHTTP     
+- NotificationEndpointPagerDuty
+- NotificationEndpointSlack    
+- NotificationRule             
+- Task                         
+- Telegraf                     
+- Variable
+
+For additional information on each resource, please see 
+[template resources](/influxdb/v2.2/influxdb-templates/#template-resources).
```

Comment:
we may want to make expand/clarify groupings being we don't have 1:1 corollary

For example, here: https://docs.influxdata.com/influxdb/v2.2/influxdb-templates/#template-resources
we see HTTP, Slack, and PagerDuty endpoint--what diff is plain `NotificationEndpoint`vs `NotificationEndpointHTTP`?

Reply:
I take back my comment here. Those resource types are import to know when using the CLI. @kelseiv The list you linked to is more of a high level list. `NotificationEndpointHTTP`, `NotificationEndpointPagerDuty`, and `NotificationEndpointSlack` are implied in "notification endpoints."